ci/test/00_setup_env_win32.sh

```diff
@@ -13,7 +13,6 @@ export PACKAGES="python3 nsis g++-mingw-w64-i686 wine-binfmt winehq-stable"
 export RUN_UNIT_TESTS=true
 export RUN_FUNCTIONAL_TESTS=false
 # export RUN_SECURITY_TESTS="true"
-export GOAL=""
+export GOAL="deploy"
 export GRIDCOIN_CONFIG="--enable-reduce-exports"
-export DEP_OPTS="NO_QT=1"
```

Comment:
is there a reason for this change?

Reply:
The artifacts should contain the Qt builds as CI is overloaded to be used for releases?